bodhi/tests/client/test___init__.py

```diff
@@ -392,6 +392,6 @@ def test_url_flag(self, send_request, query):
                 'close_bugs': True, 'stable_karma': None, 'csrf_token': 'a_csrf_token',
                 'staging': False, 'builds': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'autokarma': True,
                 'edited': u'nodejs-grunt-wrap-0.3.0-2.fc25', 'suggest': None,
-                'notes': u'this is an edited note', 'request': None, 'bugs': u'',
-                'unstable_karma': None, 'type': 'bugfix'})
+                'notes': u'this is an edited note', 'notes_file': None,
+                'request': None, 'bugs': u'', 'unstable_karma': None, 'type': 'bugfix'})
         self.assertEqual(bindings_client.base_url, 'http://localhost:6543/')
```

Comment:
@cverna  hahaha I'll remember that!

@bowlofeggs thanks so much for your advice! The click's test file thing worked wonderfully once I finally figured out how to use it. So, I added a couple of tests:

I added the two tests you mentioned 4 days ago:

> Assert that giving --notes-file properly hands the data found in a file to the notes API parameter.
> Assert that giving --notes-file and --notes prints an error.

These should also cover the tests you mentioned 2 days ago:

> Assert that passing --notes-file works in the normal case.
> Assert that passing --notes-file and --notes makes it angry.

@cverna 's test accounts for the following case you mentioned 2 days ago:

> Assert that not passing --notes-file doesn't do anything.

So I didn't add an additional test because I figured it would be redundant.

Thoughts?